transfer_nlp/plugins/helpers.py

```diff
@@ -0,0 +1,30 @@
+import torch.nn as nn
+
+from transfer_nlp.plugins.config import register_plugin
+
+
+@register_plugin
+class ObjectHyperParams:
+    """
+    Use or extend this class to configure model hyper parameters that cannot be predetermined. E.g.
+    a model input size that depends on the data set composition.
+    """
+
+    def __init__(self):
+        self.input_dim: int = None
+        self.output_dim: int = None
+
+
+@register_plugin
+class ModelParams:
```

Comment:
i was thinking this class should be renamed `TrainableParameters` or something? it makes it more clear its dealing with weights to optimize but still retains the parameters terminology from pytorch...